### ISSUE TITLE
Melhoria na flexão de gêneros

### ISSUE THREAD

**Issue by theuves** [lusofonia/extenso.js#15]
Não está passado para o feminino: *duzentos* > *duzentas*, *trezentos* > *trezentas*, e assim por diante.

Ver mais em https://pt.wikipedia.org/wiki/Dual.

**Comment by matheusgrieger** [comment]
Executando o código no RunKit para testes, se informado `gender: 'f'`, também ocorre o seguinte caso:

```
extenso('42.002', {
    number: { gender: 'f' }
});
// "quarenta e duas mil e dois"
```

O "dois" deveria ser "duas", não? Funciona se for `42.001', que fica `quarenta e duas mil e uma`.

**Comment by theuves** [comment]
Oi @matheusgrieger , perdão a demora 😀.

Hmm, fique meio confuso agora...

Você acha acredita que o correto é "quarenta e duas mil e dois"?

Vou pesquisar um pouco mais sobre isso...

Se tiver mais alguma informação, avisa aqui.

**Comment by matheusgrieger** [comment]
Não, ao contrário. Deveria ser "quarenta e duas mil e duas". Hoje a lib está trazendo como "quarenta e duas mil e dois", que acredito estar errado...

**Comment by theuves** [comment]
Certo, assim que possível vou corrgir isso.

Pull Requests serão bem-vindos.
